pkgs/development/python-modules/tqdm/default.nix

```diff
@@ -35,7 +38,7 @@ buildPythonPackage rec {
     tkinter
   ] ++
     # pandas is not supported on i686 or risc-v
-    lib.optional (!stdenv.isi686 && !stdenv.hostPlatform.isRiscV) pandas;
+    lib.optional (usePandas && !stdenv.isi686 && !stdenv.hostPlatform.isRiscV) pandas;
```

Comment:
This is only for `checkInputs` right. AFAIK, `checkInputs` are only used during build.

Reply:
Yeah:

```console
$ nix-store --query --requisites $(nix-build -A python3Packages.tqdm)
/nix/store/0z7sqj4pilbqyp45ix5b0mdgn9xlb024-libunistring-0.9.10
/nix/store/r5sjd57x0r07bwgipryaxqdkx1gglhiy-libidn2-2.3.2
/nix/store/2zchy9mdx3kk166i98cx5sriq2c1bdsn-glibc-2.33-55
/nix/store/1nmwc128408gsxb7yb3vzi0jrz0f50hh-openssl-1.1.1l
/nix/store/6kgfmzx90c1a6afqnbkz6qprkzss476k-mime-types-9
/nix/store/h22cdxm1252slxvb3qyfx1lnn85x5f0c-ncurses-6.2
/nix/store/7r1qwvpp81jbzbfdnijgvyw07p9czn5n-readline-6.3p08
/nix/store/9g6x8809cxl364lqmyi2klj8yq6d595x-zlib-1.2.11
/nix/store/ag9m9wv69898f6a1gx3sl8hamqgva537-sqlite-3.36.0
/nix/store/cqxw1wmns125k11qvrl5zkyqn9gwk5nf-expat-2.4.1
/nix/store/g0gjppf876jkk2p54v5mg20xgizns662-tzdata-2021c
/nix/store/kb1d9l6gci5m8177y4033x4lf8rhg617-libffi-3.4.2
/nix/store/lzjl6yhharpivhryidds3a4a79x4i1gd-xz-5.2.5
/nix/store/phqa311klldrcbwid1i22dwnpfc9dnma-bash-5.1-p8
/nix/store/xn86zik66mnmdgnqmd1b7qpmn0wfyibs-gdbm-1.20
/nix/store/zgdh9yy7qgvksaanb27396i5zdcpnvc7-bzip2-1.0.6.0.2
/nix/store/8dxxjbiyxwkvh53q5kh6nydla2anacgi-python3-3.9.6
/nix/store/w0qncv94l1wjmg4w9iq2yps9w2cfcfpb-python3.9-tqdm-4.62.3
```